mitodl/course-search-utils
# update linting

### What are the relevant tickets?
This is a followup to #59 to fix doof publishing, which introduced an issue with peer dependency versions. Specifically:

- #59 updated prettier to `prettier-eslint-cli@"^7.1.0"`
- But the `eslint-config-mitodl@0.2.x` was specifying `prettier-eslint-cli@"5.x"`

`yarn install` (which we use locally and on CI) works but issues some warnings; `npm install` (which doof uses) was throwing an error.

### What does this do?
This PR resolves some peer dependency conflicts. The result is that  `npm install` runs without errors.

### How can this be tested?
1. **On `main`**: Run `npm install` (not yarn!). It will error.
2. **On this branch:** Run `npm install` (not yarn!). It should succeed.
